src/libcore/iter.rs

```diff
@@ -525,11 +518,10 @@ pub trait IteratorExt: Iterator + Sized {
     /// # Example
     ///
     /// ```rust
-    /// let mut xs = range(0u, 10);
+    /// let mut it = 0 ..10;
```

Comment:
I think that it may be a typo elsewhere as well, could this be updated to `0..10`?